doc/breadthFirstSearch/pgr_binaryBreadthFirstSearch.rst

```diff
@@ -33,75 +30,82 @@ Any graph whose edge-weights belongs to the set {0,X}, where 'X' is any non-nega
+    are ignored.
+  * The returned values are ordered:
 
-  - Process is done only on 'binary graphs'. ('Binary Graph': Any graph whose edge-weights belongs to the set {0,X}, where 'X' is any non-negative real integer.)
-
-  - For optimization purposes, any duplicated value in the `start_vids` or `end_vids` are ignored.
-
-  - The returned values are ordered:
+    * `start_vid` ascending
+    * `end_vid` ascending
 
-    - `start_vid` ascending
-    - `end_vid` ascending
-
-  - Running time: :math:`O(| start\_vids | * |E|)`
+  * Running time: :math:`O(| start\_vids | * |E|)`
 
 Signatures
 -------------------------------------------------------------------------------
 
-.. code-block:: none
+.. rubric:: Summary
+
+.. parsed-literal:: 
 
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vid,  end_vid  [, directed])
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vid,  end_vids [, directed])
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vids, end_vid  [, directed])
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vids, end_vids [, directed])
-    pgr_binaryBreadthFirstSearch(Edges SQL, Combinations SQL [, directed])
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **start vid**,  **end vid**  [, directed])
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **start vid**,  **end vids** [, directed])
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **start vids**, **end vid**  [, directed])
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **start vids**, **end vids** [, directed])
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **Combinations SQL** [, directed])
     RETURNS SET OF (seq, path_seq [, start_vid] [, end_vid], node, edge, cost, agg_cost)
     OR EMPTY SET
 
-.. code-block:: none
-
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vid, end_vid)
-    RETURNS SET OF (seq, path_seq, node, edge, cost, agg_cost) or EMPTY SET
-
-:Example: From vertex :math:`2` to vertex  :math:`3` on a **directed** binary graph
-
-.. literalinclude:: doc-pgr_binaryBreadthFirstSearch.queries
-   :start-after: -- q1
-   :end-before: -- q2
+**Note:** Using the :doc:`sampledata` Network as all weights are same (i.e :math:`1``)
 
 .. index::
     single: binaryBreadthFirstSearch(One to One) - Experimental on v3.0
 
 One to One
 ...............................................................................
 
-.. code-block:: none
+.. parsed-literal:: 
 
-    pgr_binaryBreadthFirstSearch(Edges SQL, start_vid, end_vid [, directed]);
+    pgr_binaryBreadthFirstSearch(`Edges SQL`_, **start vid**, **end vid** [, directed]);
     RETURNS SET OF (seq, path_seq, node, edge, cost, agg_cost)
     OR EMPTY SET
 
-:Example: From vertex :math:`2` to vertex  :math:`3` on an **undirected** binary graph
+:Example: From vertex :math:`2` to vertex  :math:`3` on an **directed** binary graph
 
 .. literalinclude:: doc-pgr_binaryBreadthFirstSearch.queries
-   :start-after: -- q2
+   :start-after: -- q1
```

Comment:
Generate the documentation locally and verify what query is between q1 and q3 
Should be only one query.

Reply:
Never mind I see you modified the file